### PR TITLE
155432548 events bug

### PR DESCRIPTION
**Pull request by StlMaris123**
In events view, for title and description that are very long, the words surpass the text area.
The diagrams are a screen shot of the behavior before and after the fix.

screenshot from 2018-02-22 19-17-39
screenshot from 2018-02-22 16-00-33

### CODE REVIEW

**Comment by nisevi** [comment]
Hi there @StlMaris123 I think that when you mention the before and after with the screenshots it is the other way around right? Because as you mention it `before` is working just well and with your changes, meaning the `after` is breaking.

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <div class="event-box">
-  <h2><%=@event.title%></h2>
-  <p><%=@event.description%></p>
-  <p>Start date: <%=@event.start_date.strftime("%A, %B %d, %Y")%> at <%= @event.start_time.strftime("%H:%M") if @event.start_time%></p>
-  <p>End date: <%=@event.end_date.strftime("%A, %B %d, %Y")%> at <%= @event.end_time.strftime("%H:%M") if @event.end_time%> </p>
+    <div id="events_scroll">
```
Comment (nisevi) [inline app/views/events/show.html.erb]: If the changes are mean to be done over `title` and `description` you should create a class on styles for holding the behavior you want on those two elements.

Also notice that you are removing logic on the `<p>` that is not related with the fix of this PR.

**Comment by nisevi** [comment]
Notice that one of the errors you are having on [TravisCI](https://travis-ci.org/AgileVentures/LocalSupport/builds/344869965?utm_source=github_status&utm_medium=notification) :+1: 

image

```
  Scenario: User visits event directly                   # features/events/show.feature:23
    Given I visit "Open Source Weekend" event            # features/step_definitions/events_steps.rb:34
    Then I should see "Open Source Weekend"              # features/step_definitions/basic_steps.rb:341
    And I should see "Good for everyone"                 # features/step_definitions/basic_steps.rb:341
    And I should see "Sunday, October 20, 2030 at 10:40" # features/step_definitions/basic_steps.rb:341
      expected to find text "Sunday, October 20, 2030 at 10:40" in "Cookie Policy Close We use cookies to give you the best experience on our website. If you continue without changing your cookie settings, we assume that you consent to our use of cookies on this device. You can change your cookie settings at any time but if you do, you may lose some functionality on our website. More information can be found in our terms of use. Harrow Community Network Search for local voluntary and community organisations Home Organisations Login Login: Remember me Forgot your password? Sign up: New organisation? Sign up home » Events » Open Source Weekend Open Source Weekend Good for everyone Start date: Sunday, October 20, 2030 at 10:30 End date: Sunday, October 20, 2030 at 17:00 We'd love to hear your feedback here Harrow Community Network provided by Voluntary Action Harrow Co-operative&nbsp Powered by |" (RSpec::Expectations::ExpectationNotMetError)
      ./features/step_definitions/basic_steps.rb:343:in `/^I should( not)? see "([^"]*)"$/'
      features/events/show.feature:27:in `And I should see "Sunday, October 20, 2030 at 10:40"'
```

These are all the failing scenarios:
```
Failing Scenarios:
cucumber -p second_try features/admin/events/admin_new_event.feature:21 # Scenario: Successfully create an event while being signed-in as superadmin
cucumber -p second_try features/events/show.feature:23 # Scenario: User visits event directly
```
As I mentioned in the review, I think this is happening because you are removing the logic here:
```
-  <p>Start date: <%=@event.start_date.strftime("%A, %B %d, %Y")%> at <%= @event.start_time.strftime("%H:%M") if @event.start_time%></p>
-  <p>End date: <%=@event.end_date.strftime("%A, %B %d, %Y")%> at <%= @event.end_time.strftime("%H:%M") if @event.end_time%> </p>
```

This is should not be necessary since this PR is not associated with this logic, it will be enough to create a class for holding the styles :+1:

File: app/views/events/show.html.erb (hunk 1 above)
Comment (StlMaris123) [reply app/views/events/show.html.erb]: screenshot from 2018-02-22 22-31-15

Hi @nisevi it looks a little odd when the title and description are separated from start and end date. Tell me what you think

Comment (StlMaris123) [reply app/views/events/show.html.erb]: Am a little lost here which `<p>` are we talking about?

Comment (nisevi) [reply app/views/events/show.html.erb]: @StlMaris123 the ones with the `if` statement.

Comment (tansaku) [reply app/views/events/show.html.erb]: @StlMaris123 you removed `if @event.end_time` etc. did you mean to do that?

**Comment by tansaku** [comment]
@StlMaris123 you removed if @event.end_time etc. did you mean to do that?
